docs/en/latest/mtls.md

```diff
@@ -75,6 +75,14 @@ etcd:
     key: /data/certs/etcd_client.key        # path of key used by the etcd client
 ```
 
+Also set the CA certificate.
+
+```yaml
+apisix:
```

Comment:
What about just coalesce this code block and the last one (`etcd.tls`).

Reply:
put them together may be unclear